hassio/api/__init__.py

```diff
@@ -126,6 +190,18 @@ def register_snapshots(self, snapshots):
             '/snapshots/{snapshot}/restore/partial',
             api_snapshots.restore_partial)
 
+    def register_cluster(self, cluster):
+        """Register cluster function."""
+        api_cluster = APICluster(cluster, self.config, self.loop)
+
+        self.webapp.router.add_get('/cluster/info', api_cluster.info)
+        self.webapp.router.add_post('/cluster/switch_to_master',
+                                    api_cluster.switch_to_master)
+        self.webapp.router.add_post('/cluster/switch_to_slave',
+                                    api_cluster.switch_to_slave)
+        self.webapp.router.add_post('/cluster/{slug}/remove',
```

Comment:
I think it should be okay to have only `/cluster/master/{node}` we should also update  `/cluster/{slug}/remove` to `/cluser/{node}/remove`